src/components/Rerender.ts

```diff
@@ -0,0 +1,22 @@
+import { nextTick, ref } from 'vue';
+
+/**
+ * Reactive boolean that can be used to force a component to rerender.
+ */
+export const rerender = ref(true);
+
+/**
+ * Tries to rerender the component by removing it from the DOM and adding it back in.
```

Comment:
Is this good practice?

Reply:
It's the easiest way that I could think of, to share the logic between two components at the same time, and may be helpful for the future.

Reply:
alright